build.gradle

```diff
@@ -97,6 +97,7 @@ subprojects {
         protobufVersion = '3.11.4'
         protocVersion = '3.11.4'
         zipkinReporterVersion = '2.12.2'
+        zipkinVersion = '2.18.3'
```

Comment:
FYI, this isn't the latest version of ```zipkin-junit```. 
Any version of zipkin-junit newer than ```2.18.3``` use okhttp 4, which requires jvm >= 8

Reply:
Are we losing test functionality by using the older version? Should we start thinking about having integration tests that use java 8 for the testing?

Reply:
yeah it seems odd to not use java 8 in tests. Most projects have build requirements at least 8 if not above these days.. consider when you want to add features that conditionally work with futures etc.

Reply:
I agree with Adrian here, I see no reason that **all** tests shouldn't build with at least Java 8.

Reply:
I agree, but that's work for a different PR.

Reply:
I've raised #1187 to cover this